### PR TITLE
Add MMS example

### PR DESCRIPTION
**Pull request by leezu**
## Description ##
Add an example for MMS.

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1140/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1140/1/index.html

**Comment by eric-haibin-lin** [comment]
@StrayBird-ATSH  FYI

**Comment by StrayBird-ATSH** [comment]
Thank you Haibin. I will keep an eye on this existing improvement while working on this website project.



在 2020-02-06 10:54:44，"Haibin Lin" <notifications@github.com> 写道：

@StrayBird-ATSH FYI

**Comment by leezu** [comment]
@StrayBird-ATSH I don't plan to integrate this further into our website as of now. I'm mainly adding this example as it's used for https://github.com/aws/sagemaker-mxnet-serving-container

Maybe it can be useful for the website improvements. Another idea is to directly use Sagemaker API.

**Comment by StrayBird-ATSH** [comment]
@leezu So you mean currently you plan to only make this page an isolated page and it is not linked to the main website, right?

**Comment by leezu** [comment]
No, I mean I don't plan to create any page at this moment.
I'm opening this PR as the code may be helpful to you and others.

**Comment by StrayBird-ATSH** [comment]
OK. Thanks a lot for your explanation. I will have a look at it and continue on my other updates.

File: mms/README.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+==========================
+
+https://github.com/awslabs/multi-model-server/
+
+Assuming you are located in the root of the GluonNLP repo, you can run this
+example via:
+
+```
+pip install --user multi-model-server
+wget https://dist-bert.s3.amazonaws.com/demo/finetune/sst.params -O mms/sst.params
```
Comment (szha) [inline mms/README.rst]: since curl is used below, let's replace wget with curl too

**Comment by mli** [comment]
Job PR-1140/2 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1140/2/index.html

**Comment by leezu** [comment]
@szha

File: mms/README.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+https://github.com/awslabs/multi-model-server/
+
+Assuming you are located in the root of the GluonNLP repo, you can run this
+example via:
+
+```
+pip install --user multi-model-server
+curl https://dist-bert.s3.amazonaws.com/demo/finetune/sst.params -o mms/sst.params
+~/.local/bin/model-archiver --model-name bert_sst --model-path mms --handler bert:handle --runtime python --export-path /tmp
```
Comment (eric-haibin-lin) [inline mms/README.rst]: Does it path always work? `~/.local/bin/model-archiver`

File: mms/README.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+~/.local/bin/multi-model-server --start --models bert_sst.mar --model-store /tmp
```
Comment (eric-haibin-lin) [inline mms/README.rst]: would `/tmp` lead to permission issues?

File: mms/README.rst [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+curl -X POST http://127.0.0.1:8080/bert_sst/predict -F 'data=["Positive sentiment", "Negative sentiment"]'
+```
```
Comment (eric-haibin-lin) [inline mms/README.rst]: I think we should also create a page for MMS deployment on our website. But I'm fine with doing that in later PRs, not to block the release

File: mms/README.rst (hunk 3 above)
Comment (eric-haibin-lin) [reply mms/README.rst]: Does MMS auto-detect the GPU id?

File: mms/README.rst (hunk 4 above)
Comment (leezu) [reply mms/README.rst]: Agree. @StrayBird-ATSH if you like you can help with this once you have time.

File: mms/README.rst (hunk 3 above)
Comment (leezu) [reply mms/README.rst]: > would `/tmp` lead to permission issues?

Why would it?

> Does MMS auto-detect the GPU id?

No. Users would need to adapt the curl call based on MMS documentation. I haven't looked into that

File: mms/README.rst (hunk 2 above)
Comment (leezu) [reply mms/README.rst]: In a standard setup yes. This readme is currently intended as developer documentation and may need slight adjustment for non-standard setups.

File: mms/README.rst (hunk 4 above)
Comment (StrayBird-ATSH) [reply mms/README.rst]: OK. I will continue working on the GluonNLP docs website for future updates and new features.

**Pull request merged**